StantUiIosLibrary/Classes/Cards/Contacts/ContactsCard.swift

```diff
@@ -0,0 +1,242 @@
+        }
+
+        if self.callButton == nil {
+            self.callImage.set(color: .darkGrayStant)
+            self.callButton?.isEnabled = false
+        }
+    }
+
+    private func positionOptionalElements() {
+        self.addSubviews(mailImage, callImage)
+
+        self.callImage.anchor(top:      self.topAnchor,
+                              bottom:   self.bottomAnchor,
+                              trailing: self.trailingAnchor,
+                              padding:  UIEdgeInsets(top:   18, left:   0, bottom: 18,right: 16),
+                              size:     CGSize(width: 20, height: 20))
+        
+        self.mailImage.anchor(top:      self.topAnchor,
+                              bottom:   self.bottomAnchor,
+                              trailing: self.callImage.leadingAnchor,
+                              padding:  UIEdgeInsets(top: 18, left: 0, bottom: 18, right: 16),
+                              size:     CGSize(width: 20, height: 20))
+
+        if let callButton = self.callButton {
+            self.addSubview(callButton)
+
+            self.callButton?.anchor(top:      self.topAnchor,
+                                    bottom:   self.bottomAnchor,
+                                    trailing: self.trailingAnchor,
+                                    padding:  UIEdgeInsets(top: 18, left: 0, bottom: 18, right: 16),
+                                    size:     CGSize(width: 20, height: 20))
+        }
+
+        if let mailButton = self.mailButton {
+            self.addSubview(mailButton)
+
+            self.mailButton?.anchor(top:      self.topAnchor,
+                                    bottom:   self.bottomAnchor,
+                                    trailing: self.callImage.leadingAnchor,
+                                    padding:  UIEdgeInsets(top: 18, left: 0, bottom: 18, right: 16),
+                                    size:     CGSize(width: 20, height: 20))
+        }
+    }
+    
+    fileprivate func addMainViewWithShadow() {
+        mainView = UIView(frame: CGRect(x: 4, y: 2, width: self.frame.width - 8, height: self.frame.height - 4))
+        
+        guard let mainView = mainView else { return }
+        self.addSubview(mainView)
+        
+        mainView.backgroundColor = .white
+        
+        mainView.anchor(top:      self.topAnchor,
+                        leading:  self.leadingAnchor,
+                        bottom:   self.bottomAnchor,
+                        trailing: self.trailingAnchor,
+                        padding:  UIEdgeInsets(top: 2 , left: 4, bottom: 2, right: 4))
+        
+        mainView.layer.applySketchShadow(color: .lightGray, alpha: 0.2, x: 16, y: 1, blur: 0, spread: 0)
+    }
+    
+    fileprivate func configure(name: String, office: String) {
+        nameLabel    = UILabel(frame: CGRect(x: 68, y: 173, width: 210, height: 19))
+        officeLabel  = UILabel(frame: CGRect(x: 68, y: 194, width: 210, height: 14))
```

Comment:
Remove extra space from lines 138 and 139.

Reply:
Done